frontend/src/ui/components/Header/Header.tsx

```diff
@@ -1,74 +1,51 @@
-            <svg className="site-logo">
-              <use xlinkHref={`${logo}#opencraft_logo`} />
-            </svg>
-          </Navbar.Brand>
-          <Navbar.Toggle aria-controls="basic-navbar-nav" />
-          <Navbar.Collapse id="basic-navbar-nav">
-            <Nav className="ml-auto">
-              <Nav.Link>Customize</Nav.Link>
-              <Nav.Link onClick={() => window.open(CONTACT_US_LINK, '_blank')}>
-                Support Request
-              </Nav.Link>
-              <Nav.Link disabled>Status & Notifications</Nav.Link>
-              <Nav.Link
-                onClick={() => {
-                  history.push(ROUTES.Auth.LOGOUT);
-                }}
-              >
-                Log out
-              </Nav.Link>
-            </Nav>
-          </Navbar.Collapse>
-        </Navbar>
-      </Container>
-    );
-  }
+  const navigateTo = (route:string) => () => history.push(route);
 
   // Only show login link when on registration page and only show registration
   // link when on login page
   return (
     <Container>
       <Navbar bg="transparent" expand="md" variant="dark">
         <Navbar.Brand className="logo-container mx-auto">
-          <svg className="site-logo">
-            <use xlinkHref={`${logo}#opencraft_logo`} />
-          </svg>
+          <Nav.Link onClick={navigateTo(ROUTES.Console.HOME)}>
+            <svg className="site-logo">
+              <use xlinkHref={`${logo}#opencraft_logo`}/>
+            </svg>
+          </Nav.Link>
         </Navbar.Brand>
-        <Navbar.Toggle aria-controls="basic-navbar-nav" />
+        <Navbar.Toggle aria-controls="basic-navbar-nav"/>
         <Navbar.Collapse id="basic-navbar-nav">
           <Nav className="ml-auto">
-            {currentLocation.includes(ROUTES.Registration.HOME) && (
-              <Nav.Link onClick={() => history.push(ROUTES.Auth.LOGIN)}>
+            <Route path={ROUTES.Registration.HOME}>
+              <Nav.Link onClick={navigateTo(ROUTES.Auth.LOGIN)}>
                 Login
               </Nav.Link>
-            )}
-            {currentLocation.includes(ROUTES.Auth.LOGIN) && (
-              <Nav.Link onClick={() => history.push(ROUTES.Registration.HOME)}>
+            </Route>
+            <Route path={ROUTES.Auth.LOGIN}>
+              <Nav.Link onClick={navigateTo(ROUTES.Registration.HOME)}>
                 Create your account
               </Nav.Link>
-            )}
-          </Nav>
-        </Navbar.Collapse>
+            </Route>
+            <Route path={ROUTES.Console.HOME}>
+              <Nav.Link>Customize</Nav.Link>
+              <Nav.Link onClick={() => window.open(CONTACT_US_LINK, '_blank')}>
+                Support Request
+              </Nav.Link>
+              <Nav.Link disabled>Status & Notifications</Nav.Link>
+              <Nav.Link onClick={navigateTo(ROUTES.Auth.LOGOUT)}>
+                Log out
+              </Nav.Link>
+            </Route>
+          </Nav> </Navbar.Collapse>
```

Comment:
Cleaned up this page to remove duplication, and used the router to show things instead of custom code.